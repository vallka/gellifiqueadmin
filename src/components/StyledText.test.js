import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';

import { MonoText } from './StyledText';
//import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = shallow(<MonoText>Snapshot test!</MonoText>);
  expect(tree).toMatchSnapshot();
});
