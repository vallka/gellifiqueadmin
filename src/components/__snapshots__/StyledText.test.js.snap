// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component
  style={
    Array [
      undefined,
      Object {
        "fontFamily": "space-mono",
      },
    ]
  }
>
  Snapshot test!
</Component>
`;
